vxdetector/tests/test_files_manager.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python
+
+import unittest
+import os
+import tempfile
+import shutil
+import vxdetector.files_manager as fm
+
+
+class test_tmp_dir(unittest.TestCase):
+    def test_basic_function(self):
+        path = os.path.dirname(__file__)
```

Comment:
is `__file__` the sourcefile? Why do you need that? I would rely on the used library that the directory is created in an according path (typically the environment variable TMPDIR is read, ...)